api/packet_injector.go

```diff
@@ -44,14 +41,22 @@ const (
 	max = 65535
 )
 
+//PacketInjectorResourceHandler describes a packet injector resource handler
+type PacketInjectorResourceHandler struct {
+	ResourceHandler
+}
+
 // PacketInjectorAPI exposes the packet injector API
 type PacketInjectorAPI struct {
+	BasicAPIHandler
 	PIClient *packet_injector.PacketInjectorClient
 	Graph    *graph.Graph
+	Channels *packet_injector.Channels
```

Comment:
I don not see where is it used, and if so we shouldn't :)

Reply:
yes, it is used nowhere. removed!